__tests__/app.test.js

```diff
@@ -225,4 +225,44 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("status: 200 - responds with array of topic objects with comment_id, votes, created_at, author, body properties", () => {
+      const article_id = 9;
+      return request(app)
+        .get(`/api/articles/${article_id}/comments`)
+        .expect(200)
+        .then((res) => {
+          expect(res.body.comments).toHaveLength(2);
+          res.body.comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+              })
+            );
+          });
+        });
+    });
```

Comment:
Would also include a test for an empty array - article does exist but no associated comments